test/e2e/accounts/snap-account-eth-swap.spec.ts

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports */
+import { withFixtures, WINDOW_TITLES } from '../helpers';
+import { Driver } from '../webdriver/driver';
+import { accountSnapFixtures, installSnapSimpleKeyring } from './common';
+
+const {
+  buildQuote,
+  reviewQuote,
+  waitForTransactionToComplete,
+  checkActivityTransaction,
+} = require('../tests/swaps/shared');
+
+describe('Snap account swap', function () {
+  it('exchanges native token (ETH) for DAI with snap account', async function () {
+    await withFixtures(
+      accountSnapFixtures(this.test?.fullTitle()),
```

Comment:
nit: I've seen this function is used in multiple snap tests but it could be a bit misleading, as not all snap tests need the same fixtures. In fact, in this current test we don't need anything else aside from the default fixture builder and ganache. 
So having this instead of the function would simplify the testing setup:

```
      {
        fixtures: new FixtureBuilder().build(),
        ganacheOptions: defaultGanacheOptions,
        title: this.test?.fullTitle(),
      },
```


Notice how many fixtures are in the accountSnapFixtures. We don't need the test dapp server, nor the permissions or preferences controllers neither the multiple ganache options. just this:
```
{
    dapp: true,
    fixtures: new FixtureBuilder()
      .withPermissionControllerConnectedToTestDapp(false)
      .withPreferencesController({
        disabledRpcMethodPreferences: {
          eth_sign: true,
        },
      })
      .build(),
    ganacheOptions: multipleGanacheOptions,
    title,
  };
  ```

Reply:
did